src/main/resources/templates/docker/kubernetes.yml

```diff
@@ -37,8 +37,15 @@ spec:
           httpGet:
             path: {{k8sHealthCheck}}
             port: {{port}}
-          initialDelaySeconds: 5
-          periodSeconds: 5
+          initialDelaySeconds: 20
+          periodSeconds: 20
+        env:
+        - name: EUREKA_SERVER_NAME
+          value: "cloud-eureka-server"
+        - name: CONFIG_SERVER_NAME
+          value: "cloud-config-server"
+      imagePullSecrets:
```

Comment:
I don't think this is necessary

Reply:
For AKS, it may not necessary, but for all k8s users. This is the way from their docs.